extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -53,8 +59,12 @@ const TrackError = memo( ( { link, title } ) => (
 			<span>
 				<a href={ link } rel="noopener noreferrer nofollow" target="_blank">
 					<span class="jetpack-podcast-player--visually-hidden">
-						{ /* Intentional trailing space outside of the translated string */ }
-						{ /* translators: %s is the title of the track. This text is visually hidden from the screen, but available to screen readers */ }
+						{ /*
+						 * Intentional trailing space outside of the translated string.
+						 *
+						 * Translators: %s is the title of the track. This text is visually
+						 * hidden from the screen, but available to screen readers.
+						 */ }
```

Comment:
Have you tested if this comment annotation gets picked up as expected?

Reply:
It will not, but it also wouldn't in the previous form 😨. I've checked if it's being picked up from within the `{}` parens that we wrap it with inside some React renders and it turns out that it's **not** when done like this:

```js
{ /* translators: %s is the title of the track. 	*/ }
{ `${ sprintf( __( '%s:', 'jetpack' ),	title ) } ` }
```

So in such cases, we need to write it like this:

```js
{ `${ sprintf(
	// translators: %s is the title of the track.
	__( '%s:', 'jetpack' ),
	title
) } ` }
```

Just something to keep in mind.

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/15281/commits/116bc2b09cbb167da38709b588dc5e4296ed4b5f